src/Core/Components/Label/FluentLabel.razor.cs

```diff
@@ -0,0 +1,53 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// The FluentLabel component is used to display a label for an input component. Normally it is positioned above the component 
+/// </summary>
+public partial class FluentLabel
+{
+
+    /// <summary />
+    protected string? ClassValue => new CssBuilder(Class)
+        .Build();
+
+    /// <summary />
+    protected string? StyleValue => new StyleBuilder(Style)
+        .Build();
+
+    /// <summary>
+    /// Gets or sets wether the label show a required marking (red star).
+    /// </summary>
+    [Parameter]
+    public bool Required { get; set; }
+
+    /// <summary>
+    /// Gets or sets the size of the label.
+    /// </summary>
+    [Parameter]
+    public LabelSize Size { get; set; } = LabelSize.Medium;
```

Comment:
Did you tried using a nullable property, to verify is the HTML code contains the attribute or not?
If not, probably better to allow nullable property. The result will be shorter.

Reply:
Web Component always renders size and weight attributes

Reply:
I'm not talking about Web Components, but this code gives the ‘default’ values and is shorter. So it's better for the exchange between server and client: avoid including ‘extra attributes’.
In the WebComponents v3 documentation, these attributes are not mandatory either

```xml
<fluent-label>Label</fluent-label>
```

Reply:
Working on it!